package.json

```diff
@@ -36,7 +36,7 @@
     "test:debug": "mocha debug test/runner.js --reporter spec"
   },
   "dependencies": {
-    "broccoli-caching-writer": "^0.5.4",
+    "broccoli-caching-writer": "^1.1.0",
```

Comment:
Basically what this PR does, but it was changed in 2.0 to `cacheInclude` so that LESSCompiler isn't run on every build even if no less related files are changed.